controls/os_spec.rb

```diff
@@ -17,12 +17,8 @@
 # author: Dominik Richter
 # author: Patrick Muench
 
-log_dir_group = case os[:family]
-                when 'debian', 'redhat', 'fedora'
-                  'root'
-                when 'ubuntu'
-                  os[:release] == '14.04' ? 'syslog' : 'root'
-                end
+log_dir_group = 'root'
+log_dir_group = 'syslog' if os.name == 'ubuntu' && os[:release].to_i >= 14
```

Comment:
@chris-rock I'm using `os.name` here in the replacement :)

Reply:
I thought easiest just to override default of `log_dir_group = 'root'` if Ubuntu 14.04+ as the case statement seemed overkill.  If there are platforms that deviate in the future then happy to put it back.

Reply:
I agree with you. Thank you for making it simpler!

Reply:
I think we could remove version check completely as it does not really make sense (and ubuntu 16.04 has group 'syslog' as well)
```ruby
log_dir_group = os.name == 'ubuntu'? 'syslog' : 'root'
````